package.json

```diff
@@ -6,7 +6,6 @@
   "dependencies": {
     "@apollo/react-hooks": "^3.1.3",
     "@material-ui/core": "^4.9.3",
-    "@material-ui/styles": "^4.9.0",
```

Comment:
why is this deleted?

Reply:
duplicate package, that's why the styling started going crazy